.rubocop.yml

```diff
@@ -27,6 +27,10 @@ Documentation:
 Metrics/LineLength:
   Max: 120
 
+Metrics/ClassLength:
```

Comment:
I'd rather the class be refactored than silence the cop.

Reply:
It's over by one line, and I'm not experienced enough in Ruby/Rails yet to know how to refactor this well. My primary goal is to keep Greg moving

Reply:
That makes sense, but silencing the check allows for the class to grow indefinitely.

Reply:
https://jira.cru.org/browse/GT-476

Reply:
See my commit.